libs/newsletters-data-client/src/lib/draft-storage/DraftStorage.ts

```diff
@@ -11,31 +11,31 @@ export type DraftWithId = DraftNewsletterData & { listId: number };
 export const isDraft = isDraftNewsletterData;
 
 export abstract class DraftStorage {
-	abstract createDraftNewsletter(
+	abstract create(
 		draft: DraftWithoutId,
 	): Promise<
 		SuccessfulStorageResponse<DraftWithId> | UnsuccessfulStorageResponse
 	>;
 
-	abstract getDraftNewsletter(
+	abstract read(
 		listId: number,
 	): Promise<
 		SuccessfulStorageResponse<DraftWithId> | UnsuccessfulStorageResponse
 	>;
 
-	abstract modifyDraftNewsletter(
+	abstract update(
 		draft: DraftWithId,
 	): Promise<
 		SuccessfulStorageResponse<DraftWithId> | UnsuccessfulStorageResponse
 	>;
 
-	abstract deleteDraftNewsletter(
+	abstract deleteItem(
```

Comment:
Is there a reason why this is `deleteItem` rather than `delete` e.g. does `delete` get in a mess with reserved words or something?

Reply:
exactly - "delete" is a [javascript operator] (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete) so it seemed better to not use it as a method name. It probably would have been fine in most cases, but if you ever wanted to do something like ```const {create, read, update, delete} = launchService``` it would break.